packages/web-components/fast-components/src/button/README.md

```diff
@@ -0,0 +1,4 @@
+# fast-button
+`fast-button` is a web component implementation of an [HTML Button Element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button). The component supports several visual apperances (accent, lightweight, neutral, outline, stealth).
```

Comment:
nit: not sure about the capitalization of "Button" and "Element", anyone know what we should do here? It's inline so I'm inclined to think it should be "HTML button element".